dev/README.md

```diff
@@ -346,4 +346,30 @@ To create a separate solr vm create a `config.yml` file with the following conte
   external_solr:
     start: true
 ```
+# Buildkite
+
```

Comment:
Thank you for including this information! I think a better place to put this information might be in the chef-server/docs/HOW_THINGS_WORK.md.

Reply:
will make the changes